services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/ovs/DataPlaneServiceImpl.java

```diff
@@ -191,14 +189,6 @@ private void buildSubnetStates(NetworkConfiguration networkConfig, UnicastGoalSt
                 subnetConfigBuilder.setAvailabilityZone(subnetEntity.getAvailabilityZone());
             }
 
-            if (subnetEntity.getPrimaryDns() != null) {
```

Comment:
Please add //TODO comment as well as this is the v2.0 implementation.

Reply:
added.